RELEASE.md

```diff
@@ -1,27 +1,13 @@
-# *Product Model: Quantity Change For Existing Trade Lot*
+# *Updates to the Python version of CDM*
 
 _Background_
 
-This release adds support for price and/or quantity changes on an existing TradeLot, as described in issue [#2923](https://github.com/finos/common-domain-model/issues/2923).
+The current implementation of the Python version of CDM can be broken when Python restricted keywords are used in the CDM models. 
 
 _What is being released?_
 
-The `Create_QuantityChange` function has been updated to allow the price and/or quantity to be updated on an existing `TradeLot`. If the `QuantityChangeInstruction->lotIdentifier` matches the trade's `TradeLot->lotIdentifier`, then the price and/or quantity with matching units are updated based on the direction (i.e. `Increase`, `Decrease`, `Replace`) specified in the instructions.
+- A new Python generator that "mungs" Python keywords when encountered.
+- Support for two additional DSL operators (to-string, to-enum)
 
-The existing functionality is unchanged for an increase, i.e., if the `QuantityChangeInstruction->lotIdentifier` does not match the trade's `TradeLot->lotIdentifier`, then a new `TradeLot` is created.
+Depends on the completion of the [Code Generator PR #306](https://github.com/REGnosys/rosetta-code-generators/pull/306)
 
```

Comment:
How about this for a release note?:
```
# *CDM Distribution - Python Code Generation*

_What is being released?_

This release updates the `bundle` dependency to version `11.10.0` to include the new version of the Python generator which includes the following changes:

- added support for model name clashes with Python keywords, soft keywords, and items whose names begin with "_"
- added support for DSL operators `to-string` and `to-enum`
- resolves the defect exposed by PR [#2766](https://github.com/finos/common-domain-model/pull/2766)
- includes an update to the Python runtime library used to encapsulate the Pydantic support (now version 2.0.0)

_Review directions_

Download the latest Python distribution from the [Maven Central](https://central.sonatype.com/artifact/org.finos.cdm/cdm-python)

The changes can be reviewed in PR: [#2984](https://github.com/finos/common-domain-model/pull/2984)
```

Reply:
Here's the guidelines if need them https://cdm.finos.org/docs/contribution/#content-of-release-notes